### ISSUE TITLE
TestCase name with dots does not run

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#153]
```
Example Test.js

    Test = TestCase('Test.ting');
    Test.prototype.testCalled = function(){};

jstd.conf:

server: http://localhost:9876
load:
  - Test.js

Works CLI:

    java -jar jstestdriver-1.2.2.jar --config ./jstd.conf --tests all

Not Working CLI:

    java -jar jstestdriver-1.2.2.jar --config ./jstd.conf --tests Test.ing
    java -jar jstestdriver-1.2.2.jar --config ./jstd.conf --tests "Test.ing"

```

Original issue reported on code.google.com by `Olmo.Mal...@gmail.com` on 27 Oct 2010 at 2:53

**Comment by GoogleCodeExporter** [comment]
```
Oh dear. The only way to fix this is to change the syntax of the tests flag.

I'll look into options.
```

Original comment by `corbinrs...@gmail.com` on 29 Mar 2011 at 12:44

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.3.3a
```

Original comment by `corbinrs...@gmail.com` on 29 Sep 2011 at 2:59
- Changed state: **Fixed**
